### PR TITLE
Split urls.py file

### PR DESCRIPTION
**Pull request by guyy13**
Split the urls.py file into categories to avoid merge conflicts in the future.

### CODE REVIEW

File: SJMaster/urls/__init__.py
Comment (ifireball) [inline SJMaster/urls/__init__.py]: `django.urls` has an `include` function just for this.

**Review by ifireball — CHANGES_REQUESTED** [review]

File: SJMaster/urls/__init__.py
Comment (guyy13) [reply SJMaster/urls/__init__.py]: ok

Comment (guyy13) [reply SJMaster/urls/__init__.py]: done

File: SJMaster/urls/__init__.py
Comment (ifireball) [inline SJMaster/urls/__init__.py]: Why not put each file inside its own app's package?

Comment (guyy13) [reply SJMaster/urls/__init__.py]: Yea I guess that makes sense

File: SJMaster/urls/student_urls.py
Comment (ifireball) [inline SJMaster/urls/student_urls.py]: duplicate?

File: SJMaster/urls/__init__.py
Comment (guyy13) [reply SJMaster/urls/__init__.py]: done

File: SJMaster/urls/student_urls.py
Comment (guyy13) [reply SJMaster/urls/student_urls.py]: done

**Review by ifireball — APPROVED** [review]
LGTM, but other teammates must approve.
This will probably conflict with everything else when we merge it

**Review by yaelerdr — APPROVED** [review]
LGTM as well :) after its merged, well change all the urls of other PRs to look like that

**Comment by guy9050** [comment]
lgtm

**Review by guy9050 — APPROVED** [review]

**Review by eldaraharonson — APPROVED** [review]

**Review by benmenachem — APPROVED** [review]

**Review by YarinBou — APPROVED** [review]

**Pull request merged**
